go.mod

```diff
@@ -1,6 +1,12 @@
 module github.com/silinternational/certmagic-storage-dynamodb/v3
 
-go 1.17
+go 1.18
+
+require (
```

Comment:
I'm curious why you pulled these out into a separate section. (I don't mind that you did. Just wondering.)